src/main/java/ch/njol/util/VectorMath.java

```diff
@@ -106,6 +106,9 @@ public static float wrapAngleDeg(float angle) {
 		return ExprVectorFromYawAndPitch.wrapAngleDeg(angle);
 	}
 
+	/**
+	 * Copies vector components of {@code vector2} into {@code vector1}.
+	 */
 	public static void copyVector(Vector vector1, Vector vector2) {
 		vector1.copy(vector2);
 	}
```

Comment:
what's the point of this method? it's shorter to directly write `vector1.copy(vector2)`

Reply:
idk tbh vectormath is being removed anyway